### PR TITLE
Address FIXME about term for UEFI/BIOS/Firmware

### PR DESCRIPTION
**Pull request by dthaler**
A proposal for addressing a FIXME as called out in issue #84

Signed-off-by: Dave Thaler <dthaler@ntdev.microsoft.com>

### CODE REVIEW

**Comment by dthaler** [comment]
Since people didn't like the previous version of this PR, I rewrote the layered attestation section to not need to define a term at all.  Now it just uses one specific example to illustrate the concept, rather than trying to combine multiple variations in the same text.

File: draft-ietf-rats-architecture.md [hunk 1]
```diff
@@ -378,78 +372,58 @@ Attesting Environments are designed specifically with claims collection in mind.
...
+At this stage of the boot-cycle of the
+device, the Claims collected typically cannot be composed into Evidence.
+
+After the boot sequence is started, the BIOS conducts the
+most important and defining feature of layered attestation, which is that
+the successfully measured Target Environment B
+now becomes (or contains) an Attesting Environment for the next layer.
+This procedure in Layered Attestation is sometimes called "staging".
+It is important to note that the new Attesting Environment B cannot alter
+any Claims about its own Target Environment B, since those Claims were signed
```
Comment (thomas-fossati) [inline draft-ietf-rats-architecture.md:407]: When we say "signed" here are we implying a signing operation done by A in the cryptographic sense, i.e. using A's private key?  If so, I'm not sure this is the necessary precondition.  Measurements might just be "sealed" in the RoT at a certain boot stage for later use by the lead attester.

Comment (dthaler) [reply draft-ietf-rats-architecture.md]: Good point, updated.

File: draft-ietf-rats-architecture.md [hunk 2]
```diff
@@ -378,78 +372,60 @@ Attesting Environments are designed specifically with claims collection in mind.
...
+
+After the boot sequence is started, the BIOS conducts the
+most important and defining feature of layered attestation, which is that
+the successfully measured Target Environment B
+now becomes (or contains) an Attesting Environment for the next layer.
+This procedure in Layered Attestation is sometimes called "staging".
+It is important that the new Attesting Environment B not be
+able to alter any Claims about its own Target Environment B. 
+This can be ensured having those Claims be either signed by Attesting
+Environment A or stored in a shielded and secure manner by Attesting
```
Comment (dthaler) [inline draft-ietf-rats-architecture.md:409]:
```suggestion
Environment A or stored in an untamperable manner by Attesting
```

**Pull request merged**